catalog/transformer/catalog-transformer-streaming-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+           xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0
+           http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
+           http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0
+           http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd">
+
+
+    <type-converters>
+        <bean id="listConverter" class="ddf.catalog.util.impl.ListConverter"/>
+    </type-converters>
+
+    <bean id="saxEventHandlerFactoriesList" class="ddf.catalog.util.impl.SortedServiceList"/>
+    <reference-list id="saxEventHandlerFactories"
+                    interface="org.codice.ddf.transformer.xml.streaming.SaxEventHandlerFactory"
+                    availability="optional">
+        <reference-listener ref="saxEventHandlerFactoriesList" bind-method="bindPlugin"
+                            unbind-method="unbindPlugin"/>
+    </reference-list>
+
+
+    <cm:managed-service-factory id="XmlInputTransformerFactory"
+                                factory-pid="XmlInputTransformer"
+                                interface="ddf.catalog.transform.InputTransformer">
+        <service-properties>
+            <cm:cm-properties persistent-id="XmlInputTransformer"/>
```

Comment:
Did this work for setting the service properties? Also you don't need the other `cm:managed-properties` statement inside the `cm:managed-component`.